README-zh_CN.md

```diff
@@ -31,10 +31,8 @@ Eureka 是一个用户脚本，可以在任何基于 Scratch 的编辑器中加
 - [x] Creaticode
 - [x] Adacraft
 - [x] PenguinMod
-- [x] ElectraMod *
-- [x] XPLab *
-
-*\*: 仅在 CI 构建中可用*
```

Comment:
We do not consider the situation where other editors actively introduce Charlotte, and we even recommend that in this case you actively disable Charlotte in the script manager to avoid potential conflicts.

Reply:
ElectraMod and XPlab are already supported on [FurryR's commit](https://github.com/EurekaScratch/eureka/commit/7df3086759a48981e4cb5a42cde8128fb17ac3e3), i just added new TurboWarp CN editor and removed the * from XPlab and ElectraMod beause they're not only available on ci builds.